### PR TITLE
[RET-119][RET-136] File Loader for Fonts & Images

### PR DESCRIPTION
**Pull request by jcmagsay**
### What's this PR do?
- Updating `webpack.config.js` to load files into specific directories for fonts and image
- Adding fonts and font faces to reuse throughout site
- Set default font to html element

### Does this add new dependencies? Is there an installation procedure? (bundle, bower, npm, etc)
Yes, run `yarn start:dev` to (re)start server.

### What are the relevant tickets?
https://mondorobot.atlassian.net/browse/RET-119
https://mondorobot.atlassian.net/browse/RET-136

### Screenshots (if appropriate)
screen shot 2017-04-12 at 3 55 57 pm

### Who should review this PR?
@jcmagsay @mhfen

### CODE REVIEW

File: src/assets/styles/base/elements.scss [hunk 1]
```diff
@@ -9,7 +9,7 @@ html {
 
 html,
 button {
-  font-family: 'Klavika', sans-serif;
+  font-family: 'Klavika Regular', sans-serif;
```
Comment (mhfen) [inline src/assets/styles/base/elements.scss]: Use `$FONT_FAMILY_PRIMARY_REGULAR` here?

File: src/assets/styles/base/fonts.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/* ---------------------------------------------------------------------
+ Fonts
+------------------------------------------------------------------------ */
+
+@font-face {
+  font-family: 'Klavika Light';
+  src: url('../media/fonts/klavika-light.woff2') format('woff2');
```
Comment (mhfen) [inline src/assets/styles/base/fonts.scss]: I think you'll need the "Practical Level" listed here: https://css-tricks.com/snippets/css/using-font-face/#article-header-id-1

Comment (jcmagsay) [inline src/assets/styles/base/fonts.scss]: Unfortunately we don't have those fonts available, but I did add the support in comment blocks. I'll work with Geoff to get those fonts.

File: src/assets/styles/base/elements.scss (hunk 1 above)
Comment (jcmagsay) [inline src/assets/styles/base/elements.scss]: Good catch!

**Pull request merged**